server/src/test/resources/mavenWorkspace/src/main/java/com/example/App.java

```diff
@@ -0,0 +1,13 @@
+package com.example;
+
+/**
+ * Hello world!
+ *
+ */
+public class App
+{
+    public static void main( String[] args )
+    {
```

Comment:
Could you format these new Java source files with the same style as the rest of the language server? I.e.

- Opening brackets on same line
- No whitespace before and after parameters/method calls

In this case:

```suggestion
    public static void main(String[] args) {
```